tests/test_imports.py

```diff
@@ -1,12 +0,0 @@
-def test_imports():
-    """Test that most important modules can be imported
-
-    This is just a basic smoke test.
-    """
-    # ruff: noqa: F401
-
-    import aiidalab
```

Comment:
All these imports are already exercised in other tests, so we can simply delete this.